packages/backend/src/models/UserModel.ts

```diff
@@ -349,11 +349,14 @@ export class UserModel {
             lightdashUser,
             projectRoles,
         );
+        const userHasPassword = await this.hasPassword(user.user_uuid);
 
         return {
             userId: user.user_id,
             abilityRules: abilityBuilder.rules,
             ability: abilityBuilder.build(),
+            isSSO: !userHasPassword,
```

Comment:
I don't think this would work for 2 reasons:
- a user can have multiple authentication methods, user/password and sso
- we would need to have a property in the user session about what method the user used instead of checking the DB

Since we changing the onboarding to not depend on this information, should we revert all of these changes ?

Reply:
key is misleading, but also it's not needed anymore, so gonna remove it